qkeras/qrecurrent.py

```diff
@@ -161,14 +170,14 @@ def call(self, inputs, states, training=None):
       h = K.bias_add(h, quantized_bias)
 
     if rec_dp_mask is not None:
-      prev_output = prev_output * rec_dp_mask
+      quantized_prev_output = quantized_prev_output * rec_dp_mask
 
     if self.recurrent_quantizer:
       quantized_recurrent = self.recurrent_quantizer_internal(self.recurrent_kernel)
     else:
       quantized_recurrent = self.recurrent_kernel
 
-    output = h + K.dot(prev_output, quantized_recurrent)
```

Comment:
does the output include both h and c states?
Does it make sense to set the same quantizer to both h and c state?

Reply:
Just separated them in the last set of commits :)